appveyor.yml

```diff
@@ -0,0 +1,27 @@
+version: '{build}'
+
+skip_tags: true
+
+install: []
+
+environment:
+  matrix:
+    - VSVERSION: Visual Studio 12
+    - VSVERSION: Visual Studio 14
+
+platform:
+  - Win32
+  - x64
+
+configuration:
+  - Debug
+  - Release
+
+build_script:
+  - git clone https://github.com/google/googletest.git external/googletest
+  - md build
```

Comment:
Take a look at https://github.com/google/shaderc/blob/master/.appveyor.yml 
It uses cmake --build, which potentially frees up a dependency.  I also find it easier to read. :-)
Also it tests via ctest.  That will let us add other test executables without having to update this config.